stdext.opam

```diff
@@ -12,11 +12,9 @@ depends: [
   "ocaml"
   "dune" {build}
   "xapi-stdext-date"
-  "xapi-stdext-deprecated"
```

Comment:
I'm not sure these dependencies can be removed in the backport, I don't think they should be.

Reply:
I agree, I backported 40ae009014bc322856a2fafb597c94140070f55a which I think I shouldn't have. I'll just do the change manually